test/test.js

```diff
@@ -291,4 +294,16 @@ describe('notepack', function () {
 
     expect(notepack.decode(notepack.encode(fixture))).to.deep.equal(fixture);
   });
+
+  it('Infinity', function () {
```

Comment:
Yes, looks like it does. I would have expected the `NaN` one to fail but it looks like `to.deep.equal` handles that.